debug_toolbar/utils.py

```diff
@@ -401,3 +402,9 @@ def is_processable_html_response(response):
         and content_encoding == ""
         and content_type in _HTML_TYPES
     )
+
+
+class ToolbarHealthLevel(IntEnum):
```

Comment:
I think this could be called `HealthLevel` only since it actually doesn't only apply to the toolbar itself but also to individual panels.